x-pack/plugins/ml/public/application/notifications/components/notifications_list.tsx

```diff
@@ -77,7 +78,7 @@ export const NotificationsList: FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const [isLoading, setIsLoading] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
For a new setup with no jobs, I am seeing a blank page and this error in the console:

<img width="781" alt="image" src="https://user-images.githubusercontent.com/7405507/195040438-370b9527-a785-4fdd-91ff-de72ed8e8aff.png">

Reply:
Fixed in https://github.com/elastic/kibana/pull/142778/commits/6fffca893f48f8bcd186bd95320d0e90f2ccb928